api_tests/registrations/views/test_withdrawn_registrations.py

```diff
@@ -100,10 +100,10 @@ def test_withdrawn_registrations_display_limited_fields(self):
             'description': registration.description,
             'date_created': registration.date_created.isoformat(),
             'date_registered': registration.registered_date.isoformat(),
+            'date_modified': registration.date_modified.isoformat(),
```

Comment:
The build failed because `registration.date_modified` isn't rounded here, but is in the API response. Calling `registration.reload()` above should fix that.